util/src/main/java/com/farao_community/farao/util/SystematicSensitivityAnalysisService.java

```diff
@@ -36,12 +38,17 @@ private SystematicSensitivityAnalysisService() {
 
     public static SystematicSensitivityAnalysisResult runAnalysis(Network network,
                                                                   Crac crac,
-                                                                  ComputationManager computationManager) {
+                                                                  ComputationManager computationManager,
+                                                                  RaoParameters parameters) {
```

Comment:
I think it's weird to pass RaoParameters here. Either you can create like a SensitivityComputationParameters or directly pass the two information as arguments (an enum and a boolean let's say). So that you don't make it too difficult considering that it will disappear after powsybl systematic sensi is integrated !

Reply:
SensitivityComputationParameters can be nice too. I agree :heart_decoration:

Reply:
hmmm... or two new arguments as one of them is temporary